jobs/models.py

```diff
@@ -69,7 +68,81 @@ def set_expiration_date(self):
                 self.expiration_date = self.published_date + timedelta(60)
                 self.save()
 
-
     def __unicode__(self):
         return "{0}, {1}".format(self.title, self.company)
 
+
+class Meetup(models.Model):
+
+    MEETUP = 'MEET'
+    CONFERENCE = 'CONF'
+    WORKSHOP = 'WORK'
+    MEETUP_TYPES = (
+        (MEETUP, 'meetup'),
+        (CONFERENCE, 'conference'),
+        (WORKSHOP, 'workshop'),
+    )
+
+    title = models.CharField(max_length=500)
+    organization = models.ForeignKey(
+        'Company',
+        related_name="meetups",
+        blank=True,
+        null=True,
+    )
+    type = models.CharField(max_length=4, choices=MEETUP_TYPES, default=MEETUP)
+    contact_email = models.EmailField(max_length=254)
+    city = models.CharField(max_length=100)
+    country = CountryField()
+    description = models.TextField(max_length=5000)
+    is_recurring = models.BooleanField(
+        default=False,
+        help_text="Is your meetup recurring?"
+    )
+    meetup_date = models.DateTimeField(
+        null = True,
+        help_text="This stands for a starting date if the meetup is recurring"
+    )
+    reviewer = models.ForeignKey(
+        User,
+        related_name="meetups",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+    )
+    review_status = models.BooleanField(
+        default=False,
+        help_text="Check if reviewed",
+    )
+    reviewers_comment = models.TextField(
+        max_length=5000,
+        blank=True,
+        null=True,
+    )
+    ready_to_publish = models.BooleanField(default=False)
+    published_date = models.DateTimeField(blank=True, null=True)
+    created = models.DateTimeField(auto_now_add=True)
+    expiration_date = models.DateField(
+        blank=True,
+        null=True,
+        help_text="Automatically is set 60 days from posting. You can "
+                  "override this.",
+    )
+
+    class Meta:
+        unique_together = (("title", "city"),)
+        ordering = ['-published_date']
+
+    def publish(self):
+        assert self.ready_to_publish
+        self.published_date = timezone.now()
+        if not self.expiration_date:
+            self.expiration_date = self.published_date + timedelta(60)
+        self.save()
+
+    # this is temporary for admin.make_published
+    def set_expiration_date(self):
+        pass
```

Comment:
Is there any use case for having a method for setting the expiration date without publishing? I mean, the reviewer can change it manually in the admin. Do we need any other option?